### ISSUE TITLE
contact

### ISSUE THREAD

**Issue by TomanekCZ** [mliakos/farmBot#1]
Hello, is it possible to contact you due to your scripts?

thanks

**Comment by mliakos** [comment]
Yeah check out my [site](https://www.manos-liakos.info) for any info you might need.
